2024-01-17 06:14:41,249:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-01-17 06:14:41,249:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-01-17 06:14:41,249:DEBUG:certbot._internal.main:Arguments: ['--webroot', '-w', '/var/www/certbot', '--email', 'tjfgkssk9472@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'tairot.online']
2024-01-17 06:14:41,249:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-01-17 06:14:41,468:DEBUG:certbot._internal.log:Root logging level set at 30
2024-01-17 06:14:41,475:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-01-17 06:14:41,476:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f0e5330b730>
Prep: True
2024-01-17 06:14:41,477:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f0e5330b730> and installer None
2024-01-17 06:14:41,477:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-01-17 06:14:41,542:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1519022676', new_authzr_uri=None, terms_of_service=None), 053f50e92556dc2ae6fb1958ad04b4cb, Meta(creation_dt=datetime.datetime(2024, 1, 16, 8, 39, 49, tzinfo=<UTC>), creation_host='1b0b13fa81e7', register_to_eff=None))>
2024-01-17 06:14:41,544:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-01-17 06:14:41,546:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-01-17 06:14:42,052:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-01-17 06:14:42,056:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 06:14:41 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert",
  "yAgWEVtlFFc": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2024-01-17 06:14:42,061:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for tairot.online
2024-01-17 06:14:42,069:DEBUG:acme.client:Requesting fresh nonce
2024-01-17 06:14:42,069:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-01-17 06:14:42,224:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-01-17 06:14:42,226:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 06:14:41 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFs8mkLksQsOln-cDym6r4_JQ63eSS_IgpvnoPfyAYnd4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-01-17 06:14:42,226:DEBUG:acme.client:Storing nonce: 70SDU6HFs8mkLksQsOln-cDym6r4_JQ63eSS_IgpvnoPfyAYnd4
2024-01-17 06:14:42,227:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "tairot.online"\n    }\n  ]\n}'
2024-01-17 06:14:42,230:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRnM4bWtMa3NRc09sbi1jRHltNnI0X0pRNjNlU1NfSWdwdm5vUGZ5QVluZDQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "df7fxhHsVeZubdAUoCBkpQsqVF6SyOWDwdZsdHEi4PVTYeaNYSF6IPXW13K9cOIOHFvhO1lYW-Z4xipYEo1TYDMTADE5fWmhGPr2Pj6wY9xXvY2-I207rNMyz-t-oL2CuWPmKXygcesInMcp1Bn70WF2Pi3rExtqTXuvua8-Ww9m760E8ddPkqZWdDCDfSC8BhvjmGRRTgTQdrBhWFrF_Kc-0XDno9jMXbSXThSABBuHVKi_Lh_RUDaRibWzDM721znNfsoW2sy9nGa18Ia2vqmQ_D3zI4fa4ca1rg_aDz3-qynJ33cneNzltskNxqcBvMgoutov4FHg4gGgEdDNBw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogInRhaXJvdC5vbmxpbmUiCiAgICB9CiAgXQp9"
}
2024-01-17 06:14:42,419:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 339
2024-01-17 06:14:42,421:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Wed, 17 Jan 2024 06:14:41 GMT
Content-Type: application/json
Content-Length: 339
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1519022676/237291045286
Replay-Nonce: 70SDU6HF8Flzl0MlLndZJXZkuIG9dcuDrN_S9CKywr6xKLkBULE
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-01-24T06:14:41Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "tairot.online"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/305054057916"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1519022676/237291045286"
}
2024-01-17 06:14:42,422:DEBUG:acme.client:Storing nonce: 70SDU6HF8Flzl0MlLndZJXZkuIG9dcuDrN_S9CKywr6xKLkBULE
2024-01-17 06:14:42,423:DEBUG:acme.client:JWS payload:
b''
2024-01-17 06:14:42,424:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/305054057916:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICI3MFNEVTZIRjhGbHpsME1sTG5kWkpYWmt1SUc5ZGN1RHJOX1M5Q0t5d3I2eEtMa0JVTEUiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMwNTA1NDA1NzkxNiJ9",
  "signature": "daZUrJNRqwQ6PTqfc5quMDlNGdxnHyH1WbJeRXBrnryB-XnXpegQJuLn_v7Ik4aUSlozNJW9gSzGPKIsEbIk9govlp03GggO7z6UY1kn9j0HVa6U489gqzbEWStI_5dNZRXEFj6BvDzo0isyUzvfeRd5d5fhFKw03758Iss_xClytBxJMD6yRBdu156wM-3MCLhqwFOLYz0jtRcN8mlI6S-fRT_MZ1FQnBjM5T3brYr6x0fPh6NHLBUvG0_hTYLQPLV_2oS8IT7-CX603zbC0YQp8tcXJ1K-OodJSJkHZU5kQ-MiJAC3G05tBcq5nzLqKxxQecw2utaQ-GtYskhvTg",
  "payload": ""
}
2024-01-17 06:14:42,583:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/305054057916 HTTP/1.1" 200 797
2024-01-17 06:14:42,585:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 06:14:41 GMT
Content-Type: application/json
Content-Length: 797
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: LPSUY_lx4GmyvSp8Q73ilpxsRAHYeXMlxCWtNZn7hjdqFbuiV20
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "tairot.online"
  },
  "status": "pending",
  "expires": "2024-01-24T06:14:41Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/lSqERQ",
      "token": "DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/33D8mQ",
      "token": "DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/WItbxw",
      "token": "DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI"
    }
  ]
}
2024-01-17 06:14:42,585:DEBUG:acme.client:Storing nonce: LPSUY_lx4GmyvSp8Q73ilpxsRAHYeXMlxCWtNZn7hjdqFbuiV20
2024-01-17 06:14:42,587:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-01-17 06:14:42,587:INFO:certbot._internal.auth_handler:http-01 challenge for tairot.online
2024-01-17 06:14:42,588:INFO:certbot._internal.plugins.webroot:Using the webroot path /var/www/certbot for all unmatched domains.
2024-01-17 06:14:42,589:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /var/www/certbot/.well-known/acme-challenge
2024-01-17 06:14:42,596:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /var/www/certbot/.well-known/acme-challenge/DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI
2024-01-17 06:14:42,599:DEBUG:acme.client:JWS payload:
b'{}'
2024-01-17 06:14:42,601:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/lSqERQ:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICJMUFNVWV9seDRHbXl2U3A4UTczaWxweHNSQUhZZVhNbHhDV3ROWm43aGpkcUZidWlWMjAiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMwNTA1NDA1NzkxNi9sU3FFUlEifQ",
  "signature": "VceZx4E3TDDt5VMBbtp_y8Ps90QG_0SYwWOyQ5xxS0F9Wg7sLrSNkFPe0Nb5Y4shN1WUeZkPjKYRcr2Io_CQu-WigueCveTbMUK_AYeLjwz4XONqhHocul9l2pMSgw1uK-7YyDVAJ6TBRpTKphxx2GHPwnH4aS_Z7zsCZvUwjpHg0sXiUAz__Z4ssii-SQaPQ-OLnQuKE4L4HrSNB7HMctI4-6FBLIPRTY-wpcS-916tTw0Q7ojfaDuV2Jda6_pnSX85QoiCTPEOzp5VBa_zvK9XVCSQv65XVdhvZ48Yh60vT2zRjSwDbUll850pFNkDToaLafifJYEGwa1u3_8cUg",
  "payload": "e30"
}
2024-01-17 06:14:42,759:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/305054057916/lSqERQ HTTP/1.1" 200 187
2024-01-17 06:14:42,760:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 06:14:42 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/305054057916>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/lSqERQ
Replay-Nonce: LPSUY_lxod1ke5Xm7Sj6F3PtDkBVFNuHNd9dV4_FzkP2qLZ6Pkw
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/lSqERQ",
  "token": "DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI"
}
2024-01-17 06:14:42,761:DEBUG:acme.client:Storing nonce: LPSUY_lxod1ke5Xm7Sj6F3PtDkBVFNuHNd9dV4_FzkP2qLZ6Pkw
2024-01-17 06:14:42,763:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-01-17 06:14:43,764:DEBUG:acme.client:JWS payload:
b''
2024-01-17 06:14:43,768:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/305054057916:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTUxOTAyMjY3NiIsICJub25jZSI6ICJMUFNVWV9seG9kMWtlNVhtN1NqNkYzUHREa0JWRk51SE5kOWRWNF9GemtQMnFMWjZQa3ciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMwNTA1NDA1NzkxNiJ9",
  "signature": "IgcYJVhEePaQLqg0h4VNnX1OkgAVZQhav6ipdCGToPqSM-8RLr_l3AjN1ITEyCh1qYxVqaP0ebIC1qG7xhf70PSVfhBBMhhr1O4kcz2wc-Xo85j7nSu05T1hrkQCqJShhEDjGH6xr2ZDBIHkUTSoZ-IVnYMHwwPPE69mynffI-bDBV1zBCbvH01g0cmp77jXpEvD__ZIG6SkNZmFg0TuiMOjDoOeg0SXIStatPK84UDbR3qAOzmhPThl9WAhWhTxfYSRcCIwktoLCKPz4Yvql9MI2RFq3L9qHEbNodDkVKifgJXLB7c0aQ4K7lugUj-VN4GJoiM0oK20Iq2_h5U6vg",
  "payload": ""
}
2024-01-17 06:14:43,927:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/305054057916 HTTP/1.1" 200 1023
2024-01-17 06:14:43,928:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Wed, 17 Jan 2024 06:14:43 GMT
Content-Type: application/json
Content-Length: 1023
Connection: keep-alive
Boulder-Requester: 1519022676
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 70SDU6HFBHaWLFB-HcHn-eMQnNxNZgXDg6tyS7e8DaVO2CWKIL8
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "tairot.online"
  },
  "status": "invalid",
  "expires": "2024-01-24T06:14:41Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:connection",
        "detail": "43.202.208.226: Fetching http://tairot.online/.well-known/acme-challenge/DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI: Connection refused",
        "status": 400
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/305054057916/lSqERQ",
      "token": "DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI",
      "validationRecord": [
        {
          "url": "http://tairot.online/.well-known/acme-challenge/DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI",
          "hostname": "tairot.online",
          "port": "80",
          "addressesResolved": [
            "43.202.208.226"
          ],
          "addressUsed": "43.202.208.226"
        }
      ],
      "validated": "2024-01-17T06:14:42Z"
    }
  ]
}
2024-01-17 06:14:43,929:DEBUG:acme.client:Storing nonce: 70SDU6HFBHaWLFB-HcHn-eMQnNxNZgXDg6tyS7e8DaVO2CWKIL8
2024-01-17 06:14:43,930:INFO:certbot._internal.auth_handler:Challenge failed for domain tairot.online
2024-01-17 06:14:43,930:INFO:certbot._internal.auth_handler:http-01 challenge for tairot.online
2024-01-17 06:14:43,931:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: tairot.online
  Type:   connection
  Detail: 43.202.208.226: Fetching http://tairot.online/.well-known/acme-challenge/DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI: Connection refused

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-01-17 06:14:43,932:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-01-17 06:14:43,934:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-01-17 06:14:43,934:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-01-17 06:14:43,935:DEBUG:certbot._internal.plugins.webroot:Removing /var/www/certbot/.well-known/acme-challenge/DO_rrKEZB5ZXb0hvtPiLqU0pGIzvMPdZ66uInx5CLGI
2024-01-17 06:14:43,939:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-01-17 06:14:43,940:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-01-17 06:14:43,941:ERROR:certbot._internal.log:Some challenges have failed.
